core/templates/components/forms/custom-forms-directives/thumbnail-uploader.component.ts

```diff
@@ -102,6 +102,8 @@ export class ThumbnailUploaderComponent implements OnInit, OnChanges {
           this.contextService.getEntityId()));
       this.uploadedImage = this.editableThumbnailDataUrl;
       this.thumbnailIsLoading = false;
+    } else {
+      this.hidePlaceholder = true;
```

Comment:
Why is this required?

Reply:
i don't know Mridul had added this removed